data/categories.txt

```diff
@@ -6061,7 +6063,7 @@ nb:Vår
 pl:Źródło
 pt:Nascente
 pt-BR:Nascente
-ro:Primăvară
+ro:Izvor
```

Comment:
There is also data/strings/types_strings.txt with types translations. It would be better if here, in categories, will be at least the same string that is used in types_strings.txt.
We plan to refactor to avoid duplication in the future, but now, unfortunately, there is no other way.

Reply:
> There is also data/strings/types_strings.txt with types translations. It would be better if here, in categories, will be at least the same string that is used in types_strings.txt (plus some more synonyms that people can type). We plan to refactor to avoid duplication in the future, but now, unfortunately, there is no other way.

okay, so if I understand correctly, I need to change the types_strings.txt words to match the first word of categories.txt?

Reply:
types_strings contains texts that are _visible_ to users. Users expect that these _visible_ types (we sometimes call them "categories") will also be searchable. That's why the translation from types_strings should be duplicated in categories.txt (but only if this type should be _searchable_ by users).